.github/workflows/.reusable-cleanup-registry.yml

```diff
@@ -25,8 +24,7 @@ jobs:
           untagged-only: true
```

Comment:
my linter says, this param doesn't exist anymore 🤷

Reply:
You're right. I only fixed the ones that came up in the failed pipeline job. Fixed another few